### PR TITLE
Add RETURN documentation

### PR DESCRIPTION
**Pull request by bcarroll**
Initial RETURN documentation created

+label: docsite_pr

##### SUMMARY
Added RETURN documentation

RETURN documentation was empty

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
file

##### ADDITIONAL INFORMATION
No code enhancements, simply a documentation addition

### CODE REVIEW

File: lib/ansible/modules/files/file.py [hunk 1]
```diff
@@ -173,8 +173,87 @@
     modification_time: now
     access_time: now
 '''
-RETURN = '''
 
+RETURN = r'''
+changed:
```
Comment (bcoca) [inline lib/ansible/modules/files/file.py:178]: changed/failed/skipped are 'common return fields' and don't need to be documented  in each plugin

Comment (acozine) [reply lib/ansible/modules/files/file.py:178]: @bcarroll common return values are documented [here](https://docs.ansible.com/ansible/latest/reference_appendices/common_return_values.html#common-return-values)
When you build your module docs, the `RETURN` section should include a pointer to that page of common return values documentation just under the header. You can test this locally - follow the instructions at https://docs.ansible.com/ansible/devel/dev_guide/testing_documentation.html.

File: lib/ansible/modules/files/file.py [hunk 2]
```diff
@@ -173,8 +173,87 @@
...
+  description: Textual gid
+  returned:
+  type: string
+  sample: appusers
+mode: 
+  description: mode of dest
+  returned:
+  type: string
+  sample: '0777'
+owner: bcarroll
```
Comment (acozine) [inline lib/ansible/modules/files/file.py:231]: did you mean this as a `sample`?

**Comment by acozine** [comment]
@bcarroll thanks for dedicating a lot of time and effort to documenting the return values of the `file` module. This PR needs to pass our CI system before we can review it again. If you need help getting your branch to pass the sanity tests, feel free to reach out on the #ansible-docs channel on freenode IRC.

**Comment by bcarroll** [comment]
Thanks for all the feedback and the reference documentation.  I will go through that process before doing any further pull requests.

**Pull request closed without merging**